bagel/tests/test_utility.py

```diff
@@ -352,25 +357,29 @@ def test_invalid_age_heuristic():
     assert "unrecognized age transformation: nb:birthyear" in str(e.value)
 
 
+# TODO: See if we can remove this test: it's a little hard to maintain and
+# essentially replicates the logic of the function being tested
+# Instead, see test_all_used_namespaces_have_urls and test_used_namespaces_in_context
 @pytest.mark.parametrize(
     "model, attributes",
```

Comment:
Good point! Could you make an issue for that?

Reply:
will address as part of https://github.com/neurobagel/bagel-cli/issues/350!